app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: :User
+  belongs_to :post
+
+  def update_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
## Optional
- [ ] I highly suggest that you use [Rails Active Callback](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_comment_counter` method every time a new post has been saved to the database, then make the `update_comment_counter` method private